hey.go

```diff
@@ -135,6 +135,10 @@ func main() {
 	// set content-type
 	header := make(http.Header)
 	header.Set("Content-Type", *contentType)
+
+	// set a default user-agent
+	header.Set("User-Agent", "hey (https://github.com/rakyll/hey)")
```

Comment:
Is it possible to keep the Go user agent by appending to header.Get("User-Agent"). It could be good that the user agent was just appending, more like:

`Go-http-client/1.1 hey/master (github.com/rakyll/hey)`

I'd double check the RFC to see `.` or `/` is allowed in the comment section of the user-agent.